data/hails.txt

```diff
@@ -2411,7 +2520,6 @@ phrase "hostile navy"
 	word
 		"."
 
-
```

Comment:
Why the deletion? Additonal spaces are used to help with conceptual isolation. The game between two dissimilar phrases (e.g. "hostile navy" and "friendly deep" is larger that the difference between "friendly deep" and "friendly deep"
(See the style guide from the wiki)

Reply:
It still has the spacing, but tbqh it was a mistake to remove it